spec/policies/group_policy_spec.rb

```diff
@@ -42,7 +42,7 @@
 
     context "and in the same organisation as the group" do
       it "forbids only request_upgrade, review_upgrade, delete and destroy" do
```

Comment:
I think for this and the other test you've edited it would be useful to add `move` to the test name:
```suggestion
      it "forbids only request_upgrade, review_upgrade, delete, destroy and move" do
```